devenv/local/.gitignore

```diff
@@ -0,0 +1,2 @@
+K8S
+CUSTOM-K8S-DOCKER-BUILDS
```

Comment:
Can you move this (and the .gitignore inside docker-compose) into the top-level `.gitignore`? That way it's much easier to debug where/why folders are ignored

Reply:
I've removed the extra `.gitignore` files and changed md file names